### ISSUE TITLE
Making Contributing to the Evangelism WG Accessible and Easy

### ISSUE THREAD

**Issue by bnb** [nodejs/evangelism#26]
Apparently people want to contribute to the Evangelism WG, and don't know how. What can we do to make this obvious and easy? If you have _any_ ideas, please share them. Also **if you're trying to find a way to contribute**, participate in this conversation! We want to hear from you, and help you get involved immediately.

Right now, we don't have a lot of outlets for people to create content for. What are some ideas for this? Official channels and community channels are both possibilities. We _need_ to come up with several ways for people to help out.

Just a few ideas I have: 
- Document creation: Create issues/PRs for documents that we can use to help the community
- Document addition: There are several additions to the Social Media Style Guide I'd like to make, but haven't had time for. I'm going to create an issue for this, so someone can help work on that.
- Community Blog Posts: Community posts to the blog, highlighting specific aspects of the community, tools, companies, etc.

**Comment by dotproto** [comment]
@bnb what sections do you want to add to the style guide? A quick bulleted list would be quite helpful :smile: 

It seems like we're doing a regular "io.js Week of X" blog on Medium. Where is that being drafted? Are there any other regular posts we're writing up? We may want to create .MD file that lists regular activities and links to the appropriate folders in the evangelism/website repo.

**Comment by bnb** [comment]
@svincent The sections I wanted to add to the style guide were the things we talked about in our Hangout. Unfortunately, while I was using git I deleted the file, and apparently I didn't copy it to a backup folder like I thought I did. The only section I wanted to add that I can remember was the special markdown formatting for long form posts - there were several others, however.

**Note:** I just noticed that I said I'd create an issue for those updates to the style guide. I don't think I'll be able to do that now, since I don't have the list anymore.

For the weekly updates, those are being drafted as issues in this repo. https://github.com/iojs/evangelism/issues/16 is the current one, https://github.com/iojs/evangelism/issues/4 is last week's. Previous weeks' were in the same issue format in iojs/website.

Right now, the weekly update is the only regular content we have. The goal of this issue was to try to create more regular content and such that could be written by community members.
